test/test_projective.jl

```diff
@@ -34,12 +34,20 @@ gpt2 = [100.0, -200.0, 0.0]
 (xy, visible) = cameraproject(Cam, [gpt1 gpt2], computevisibility=true)  # project to image
 
 # Then project image points to ground plane to see if we reconstruct them
-# ** Note this does not seem to work with lens distortion
 planeP = [0.0,0.0,0.0]
 planeN = [0.0,0.0,1.0]
 planept = imagept2plane(Cam, xy, planeP, planeN)
 @test maximum(abs.(planept - [gpt1 gpt2])) < f*tol
 
+#same test as above with radial distortion
+Cam.k1=-0.2
+Cam.k2=0.2
+Cam.k3=-0.2
+
+xy = cameraproject(Cam, [gpt1 gpt2])
+planept = imagept2plane(Cam, xy, planeP, planeN,:iterative)
```

Comment:
Thanks for the work however the tests are failing on this line as you are calling imagept2plane() with the sysmbol :iterative.  Is this intended?

Reply:
Oh right! During my testing I identified the method to be used for the distortion inversion with a keyword. But since at the end only one method was left, the keyword was removed entirely. I seem to have forgotten about doint it in the test also. I fixed it in [7111daf](https://github.com/peterkovesi/ImageProjectiveGeometry.jl/pull/32/commits/7111daf6a058a4e9c83919e9206c47a3c566498d) and created copies of the Camera instance and coordinates to not mess with the next tests.